04/style.css

```diff
@@ -0,0 +1,160 @@
+* {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+    font-family: Arial, Helvetica, sans-serif;
+}
+body {
+    background-color: #efeff5
+    ;
+}
+a, a:visited, a:focus, a:hover, a:active { text-decoration: none; color:#000; }
+    a:hover {color:black}
+.hamburger {
+    width: 40px;
+    height: 35px;
+    border: 0;
+    border-top: 5px black solid;
+    background: transparent;
+    position: relative;
+    outline: none;
+    transition: 0.3s;
+    cursor: pointer;
+}
+.hamburger::after, .hamburger::before {
+    content: "";
+    width: 100%;
+    top: 0;
+    left: 0;
+    position: absolute;
+    border-top: 5px solid black;
+    transform: translateY(10px);
+}
+.hamburger::after {
+    transform: translateY(25px);
+    transition: 0.3s;
+}
+.hamburger:focus {
+    transform: rotate(45deg) translateY(5px);
+    border: 0;
+}
+.hamburger:focus::after {
+    transform: rotate(-90deg) translateX(-10px);
+}
+/*jak zaimplementowac bem zaczynajac od mobile */
```

Comment:
Nie do końca rozumiem problem. BEM to m. in. nazewnictwo klas i nie ma związku z RWD